pootle/templates/browser/overview.html

```diff
@@ -78,13 +78,21 @@ <h4 class="clear">{{ item.title }}</h4>
 {% if has_sidebar %}
 <div id="sidebar"
   class="js-sidebar{% if is_sidebar_open %} sidebar-open{% endif %}">
-  {% block sidebar_content %}
+  <div id="sidebar-content">
+    {% block sidebar_content %}
 
-  {% if announcement %}
-  {% include "staticpages/_body.html" with page=announcement %}
-  {% endif %}
+    {% if announcement %}
+    {% comment %}Dirty hack to place a separator while we don't have a way to
+    tell if after an existing item we have another one.{% endcomment %}
+    {% if display_download or upload_form %}
```

Comment:
Having said that, it's still possible to add the separator by having the content for each content type within a block, and using CSS have all those blocks, except the last one, to render a `border-bottom`.